Resources/Documentation/installation_and_setup.md

```diff
@@ -34,7 +29,18 @@ brew upgrade
 ```
 (This command will upgrade all of the programs you've installed through Homebrew.)
 
-Thus far installing vrecord on Linux using Linuxbrew has not been successful.
+### Linux
+
+As of version 2019-01-19, vrecord can now be installed on select Linux distributions, most easily via [Linuxbrew](https://docs.brew.sh/Homebrew-on-Linux), with the same macOS Homebrew Terminal commands:
```

Comment:
Think it isn't technically called `Linuxbrew` any more!

Reply:
Indeed, the document begins now with the sentence:

> The Homebrew package manager may be used on Linux and Windows Subsystem for Linux (WSL).